File: app/src/main/java/hu/pe/biko/biko/BikoApi.java
package hu.pe.biko.biko;

import java.util.List;

import io.reactivex.Flowable;
import retrofit2.http.GET;

/**
 * Created by Galya Sheremetova on 27.05.2017.
 */

public interface BikoApi {
    @GET("routes")
    Flowable<List<Route>> getRoutes();

    @GET("places")
    Flowable<List<Place>> getPlaces();
}

File: app/src/main/java/hu/pe/biko/biko/BikoApi.kt
package hu.pe.biko.biko

import io.reactivex.Flowable
import retrofit2.http.GET

/**
 * Created by Galya Sheremetova on 27.05.2017.
 */

interface BikoApi {
    @get:GET("routes")
    val routes: Flowable<List<Route>>

    @get:GET("places")
    val places: Flowable<List<Place>>
}
